src/index.html

```diff
@@ -160,6 +167,37 @@ <h2 class="modal__title">New post</h2>
         </div>
       </div>
 
+      <div id="avatar-modal" class="modal">
+        <div class="modal__container">
+          <h2 class="modal__title">Edit profile avatar</h2>
+          <button
+            type="button"
+            id="modal__close-button-avatar"
+            class="modal__close-button"
+          ></button>
+          <form name="add-avatar" id="add-avatar" class="modal__form">
+            <label for="image-link" class="modal__label">
```

Comment:
`for="image-link"` os a;readu om ;ome 135, so you need to use another `id` and `form` for the  `avatar` modal


You can use `avatar-link`

Reply:
* It's necessary to fix all the errors (only **errors**, not warnings) found by html `validator`: https://validator.w3.org/nu/#textarea . Just copy your HTML code to the text area and hit `Check`. This validator will help you find and fix small typos, syntax errors and other similar issues. This short **video** shows how to use validators for your code: https://youtu.be/Yc_Qw4-oVq0